tests_integration/test_stage.py

```diff
@@ -19,6 +19,7 @@
 
 import pytest
 
+from tests_common import IS_WINDOWS
```

Comment:
This doesn't seem to be used in this file 😅

Reply:
Fixed